src/nvme/mi-mctp.c

```diff
@@ -84,6 +86,9 @@ struct nvme_mi_transport_mctp {
 	int	sd;
 	void	*resp_buf;
 	size_t	resp_buf_size;
+	int		sd_aem;
```

Comment:
alignment seems odd here

Reply:
Still seems odd; `sd_aem` looks to be one indent too much?

(super minor, but still)

Reply:
Still incorrect alignment. You have:

```c
	void	*resp_buf;
	size_t	resp_buf_size;
	int		sd_aem;
```

you want:

```c
	void	*resp_buf;
	size_t	resp_buf_size;
	int	sd_aem;
```

There are other instances of the same thing elsewhere too. I won't worry about this for review, but it's still a minor niggle.